### PR TITLE
Added Settings screen + settingsReducers modifications

### PR DESCRIPTION
**Pull request by aatalyk**

### CODE REVIEW

File: src/components/services/ServicesList.js [hunk 1]
```diff
@@ -1,86 +1,100 @@
...
-      </View>
-    );
-  }
+	render() {
+		const { isPartiallyShown } = this.props;
+		return (
+			<View style={[styles.container, this.props.style, { flex: !isPartiallyShown ? 1 : undefined }]}>
+				{isPartiallyShown && <Text style={styles.title}>Services</Text>}
+				<View>
+					<FlatList
```
Comment (aarkalyk) [inline src/components/services/ServicesList.js]: Please, add a `keyExtractor`.

File: src/components/settings/Settings.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	onValueChange = () => {
+		console.warn(this.props.notifsEnabled);
+		this.props.notifsEnabled ? this.props.disableNotification() : this.props.enableNotification();
+	};
+
+	render() {
+		return (
+			<View style={styles.container}>
+				<LanguageItem title="Қазақша" onPress={this.onPress} isSelected={this.props.lang === 'kaz'} />
```
Comment (aarkalyk) [inline src/components/settings/Settings.js]: How about adding a title here. e.g. "Косымша тили/язык приложения"

Comment (aatalyk) [reply src/components/settings/Settings.js]: No need I think

File: src/logic/newsLogic.js [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
 
 import { FETCH_NEWS_REQUESTED, fetchNewsSucceeded, fetchNewsFailed } from '../actions';
-import { database } from '../config';
+import { fetch } from '../config';
 
 const fetchNewsLogic = createLogic({
-  type: FETCH_NEWS_REQUESTED,
-  warnTimeout: 0,
-  process: async (_, dispatch) => {
-    database.ref('news').on(
```
Comment (aarkalyk) [inline src/logic/newsLogic.js]: we need to discuss this one later. according to the firebase docs, this reacts to changes in realtime database. By using it as a one time function as we do now, we might be missing out on a cool feature.

File: src/scenes/Home.js [hunk 4]
```diff
@@ -13,92 +13,92 @@ import { Notifications } from './Notifications';
...
 
 const propTypes = {
-  navigation: PropTypes.object,
-  newsItems: PropTypes.array,
-  fetchNewsRequested: PropTypes.func,
-  lang: PropTypes.string,
+	navigation: PropTypes.object,
+	newsItems: PropTypes.array,
+	fetchNewsRequested: PropTypes.func,
+	lang: PropTypes.string
```
Comment (aarkalyk) [inline src/scenes/Home.js]: I think we need a custom `PropType` for `lang`. 

```Javascript
PropTypes.oneOf(['kaz', 'rus'])
```
This will be much safer.

Comment (aarkalyk) [reply src/scenes/Home.js]: what about this?

Comment (aatalyk) [reply src/scenes/Home.js]: Need more clarification

Comment (aarkalyk) [reply src/scenes/Home.js]: `lang` should not be a plain string, it should be an enum, either `"rus"` or `"kaz"` nothing else.  
```Javascript
lang: PropTypes.oneOf(['kaz', 'rus'])
```
better to move it to `propTypes/index.js` and import from there.

**Pull request merged**